tests/platforms/kubernetes/appmanager.go

```diff
@@ -234,6 +235,7 @@ func (m *AppManager) CreateIngressService() (*apiv1.Service, error) {
 
 // AcquireExternalURL gets external ingress endpoint from service when it is ready
 func (m *AppManager) AcquireExternalURL() string {
+	log.Printf("Waiting until service has reached target state...\n")
```

Comment:
Just for debugging? If so, revert file changes.

Reply:
Given that "deployment" is often where things fail (in general) I think it's good to log around the time when such an operation starts, otherwise we spent time instead trying to narrow down when the problem occurred.

Reply:
because presumably this is called right after deploying